api/api.go

```diff
@@ -315,3 +315,33 @@ type ArchiveOpenOptions struct {
 type ArchiveCreateOptions struct {
 	LogSizeThreshold *int64 `json:"log_size_threshold,omitempty"`
 }
+
+// AuthAPIAudience is the audience claim for any zqd api access token.
+// Though represented as a URL, it does not need to be a reachable location.
+const AuthAPIAudience = "https://app.brimsecurity.com"
+
+type AuthIdentityResponse struct {
+	TenantID string `json:"tenant_id"`
+	UserID   string `json:"user_id"`
+}
+
+type AuthMethod string
+
+const (
+	AuthMethodNone  AuthMethod = ""
+	AuthMethodAuth0            = "auth0"
```

Comment:
I'm pretty sure this pattern only works with iota:
```suggestion
	AuthMethodAuth0 AuthMethod = "auth0"
```

Reply:
You're correct, thanks: https://golang.org/ref/spec#Constant_declarations